src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -11572,23 +11572,31 @@ definitions:
       value:
         description: The value for the given setting
         type: object
-  RulesExecution:
+  RuleExecution:
     required:
-      - includeSubfolders
+      - hasSubFoldersIncluded
+      - hasInheritedRulesExecuted
     properties:
-      includeSubfolders:
-        description: Whether the rules will also run against subfolders
+      hasSubFoldersIncluded:
+        description: Whether the rules were executed also against sub-folders
         type: boolean
-  RulesExecutionEntry:
+      hasInheritedRulesExecuted:
+        description: Whether the inherited rules were also executed
+        type: boolean
+  RuleExecutionEntry:
     type: object
     required:
       - entry
     properties:
       entry:
-        $ref: '#/definitions/RulesExecution'
-  RulesExecutionBody:
+        $ref: '#/definitions/RuleExecution'
+  RuleExecutionBody:
     properties:
-      includeSubfolders:
-        description: Run the rules against the given folder and all subfolders
+      hasSubFoldersIncluded:
+        description: Execute rules against the given folder and all sub-folders
+        type: boolean
+        default: false
+      hasInheritedRulesExecuted:
```

Comment:
Maybe `areSubfoldersIncluded` and `areInheritedRulesExecuted`?

Reply:
That was my first thought, but I wasn't sure because:
1. I would like to keep same parameter names in input and output (one Java model).
2. It's a way of interpretation, since we might say: "Add rule execution which has sub-folders included" - here "are" doesn't fit.
In the other hand "are" seems to be more natural, so I doesn't insist on "has".

Reply:
I was comparing it with several other booleans where we start with "is", but I agree that it's not obvious which is best.  I'm happy with this given that you've considered the options.

Reply:
That one isn't easy. Maybe `shouldIncludeSubFolders` / `shouldExecuteInheritedRules` ? But go with the choice you find best.